framework/core/src/Mail/MutateEmail.php

```diff
@@ -15,10 +15,10 @@ class MutateEmail
 {
     public function handle(MessageSending $event): bool
     {
-        if (! empty($link = $event->data['unsubscribeLink'])) {
+        if (! empty($event->data['unsubscribeLink'])) {
```

Comment:
```suggestion
        $unsubscribeLink = Arr::get($event->data, 'unsubscribeLink');
        if (! $unsubscribeLink) {
```

Can't we use something like this for better code readability?

Imports not loaded nor variable added in line 21.